pages/prachi-chauhan.md

```diff
@@ -21,6 +21,6 @@ Should you have any queries on my profile, please feel free to reach out to me v
 * prachichauhan777@gmail.com
 * pxc163630@utdallas.edu
 * contact number: (972)602-5071
-* linked in profile: <https://www.linkedin.com/in/prachi-chauhan-utd20162018/>
+* [linked in profile]<https://www.linkedin.com/in/prachi-chauhan-utd20162018/>
```

Comment:
Change `[linked in profile]<https://www.linkedin.com/in/prachi-chauhan-utd20162018/>` to `[linked in profile](https://www.linkedin.com/in/prachi-chauhan-utd20162018)` if you want the link to work